examples/src/main/r/0-getting-started.R

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
We need to have the Apache License at the top of every file. You can see https://github.com/apache/spark/blob/master/examples/src/main/r/dataframe.R#L1 for an example

Also per our style guide we don't put in Author names / dates in the file itself as this is tracked in the commit log